2018-pycon-russia/MLonCode.md

```diff
@@ -0,0 +1,28 @@
+# Machine Learning on Source Code
+
+**Updates**:  https://github.com/src-d/conferences/issues/49
+**Sent on**:  2018/03/20
+**Status**:   submitted
+**Author**:   Vadim Markovtsev
+**Slides**:   TBA
+**Proposal**: N/A
+
+**Abstract**:
+
+Machine Learning on Source Code (MLoSC) is an emerging and exciting domain of research which stands at the sweet spot between deep learning, natural language processing, social science and programming. The list of MLoSC resources - [awesome-machine-learning-on-source-code](https://github.com/src-d/awesome-machine-learning-on-source-code) - has already attracted more than 1,600 watchers on GitHub.
```

Comment:
MLonCode

Reply:
I bet I used the correct hashtag in the actual submission and this is just a copy-paste bug